packages/api/src/model-input.ts

```diff
@@ -1,10 +1,7 @@
-import { Model } from './index'
-
 /**
  * A generalized description of the input to an epidemiological model.
  */
 export interface ModelInput {
```

Comment:
nit: Might be nice if you exported `ModelInput` directly through `index.ts`. This way, when you reference it later you don't have to go through the `input` hash. See my comment below.